src/borg/testsuite/repository.py

```diff
@@ -328,11 +327,11 @@ def test_flags_persistence(repo_fixtures, request):
 
 
 def _assert_sparse(repository):
-    # The superseded 123456... PUT
+    # the superseded 123456... PUT
```

Comment:
Most comments started with lowercase, so for consistency I made them all uniform lowercase.